torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -2503,7 +2628,8 @@ Tensor TensorExprKernel::computeValue(const torch::jit::Value* v) {
   if (NNCLoweringFunction custom_lowering = getCustomLoweringFor(op)) {
     return custom_lowering(argInputs, outputShape, outputType, device_);
   }
-  return computeOperandValue(op, argInputs, outputShape, outputType, device_);
+  return computeOperandValue(
+      op, argInputs, outputShape, outputType, device_, name);
```

Comment:
How about we set the name of the buffer returned by `computeOperandValue` here? That would mean that we only need to change one place.

Reply:
Haha, that is literally just 1 line change. Thanks for pointing that out!